src/state/selection/types.ts

```diff
@@ -87,3 +87,8 @@ export interface SetSelectedUIDisplayDataAction {
     payload: UIDisplayData;
     type: string;
 }
+
+export enum ColorSettings {
```

Comment:
nit: make this singular. So when it's used `colorSetting.Default` it makes sense. it's a single setting

Reply:
I don't have a good suggestion for this name, but it could be confusing because at first glance it seems like it's going to be a list of colors. the singular will help, but something like, `colorsToUse` or `colorsOrigin`?